demo-app/client/test-swaggers/test-swagger.json

```diff
@@ -5,7 +5,7 @@
     "version": "v1"
   },
   "host": "example.com",
-  "basePath": "/swagger",
+  "basePath": "",
```

Comment:
It can be as you want but remember: basePath is the path to the API not the path to the swagger file. Actually the swagger file does some times not need to be served at all (when the api doc is hidden from external users).
In this case I set basePath to `""` cause I wanna the url look as 'api/...' not 'swagger/api/....'

Reply:
Please, fill it. Once we are reflecting it (yes, in api calls' url), we should also test it works.